src/openforms/js/components/admin/form_design/FormStepDefinition.js

```diff
@@ -373,6 +399,32 @@ ConfigurationErrors.propTypes = {
   errors: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string)),
 };
 
+const getDuplicatedComponents = componentNameSpace => {
+  let keyList = [];
+  let duplicates = [];
+
+  componentNameSpace.forEach(component => {
+    if (!keyList.includes(component.key)) {
+      keyList.push(component.key);
+      return;
+    }
+
+    if (!duplicates.some(duplicate => duplicate.key === component.key)) duplicates.push(component);
+  });
+
+  return duplicates;
+};
+
+const getComponentLocations = (formSteps, component) => {
```

Comment:
seems like only `component.key` is used, so perhaps that should be passed rather than the entire component?